### PR TITLE
Jp5 2d457 1 des

### PR DESCRIPTION
**Pull request by dmipx**
update for compatibility with development platform

### CODE REVIEW

**Comment by dmipx** [comment]
This was tested on JP5 single and dual camera and on JP4

File: apply_patches.sh [hunk 1]
```diff
@@ -3,24 +3,35 @@
...
     echo "apply_patches.sh reset [JetPack_version]"
     exit 1
 fi
 
-# Default to dual camera DT for JetPack 5.0.2
-JP5_D4XX_DTSI="tegra194-camera-d4xx.dtsi"
+# Default to single camera DT for JetPack 5.0.2
+# single - dev board
+# one/dual - evb
+JP5_D4XX_DTSI="tegra194-camera-d4xx-dual.dtsi"
```
Comment (jnzw) [inline apply_patches.sh:14]: The comment on line 11 says default to single camera, but on line 14 it looks like the dual dtsi is the actual default?

File: apply_patches_ext.sh [hunk 2]
```diff
@@ -2,23 +2,34 @@
...
+    echo "apply_patches_ext.sh [--one-cam | --dual-cam] source_dir [JetPack_version]"
     exit 1
 fi
 
-# Default to dual camera DT for JetPack 5.0.2
-JP5_D4XX_DTSI="tegra194-camera-d4xx.dtsi"
+# Default to single camera DT for JetPack 5.0.2
+# single - dev board
+# one/dual - evb
+JP5_D4XX_DTSI="tegra194-camera-d4xx-dual.dtsi"
```
Comment (jnzw) [inline apply_patches_ext.sh:12]: Same issue here?

File: apply_patches.sh (hunk 1 above)
Comment (dmipx) [reply apply_patches.sh]: ok done.

File: apply_patches_ext.sh (hunk 2 above)
Comment (dmipx) [reply apply_patches_ext.sh]: done.

**Comment by dmipx** [comment]
@jnzw the JP4 d4xx driver is not aligned to new cool serdes management of JP5. can we merge this feature to jp4?

**Comment by jnzw** [comment]
> @jnzw the JP4 d4xx driver is not aligned to new cool serdes management of JP5. can we merge this feature to jp4?

I suppose you mean the dual D457 feature (#144) for JP4? I think it's doable, but it will require non-trivial effort to implement and validate. The d4xx and serdes drivers diverged quite a bit between JP4 and JP5.

**Comment by dmipx** [comment]
> > @jnzw the JP4 d4xx driver is not aligned to new cool serdes management of JP5. can we merge this feature to jp4?
> 
> I suppose you mean the dual D457 feature (#144) for JP4? I think it's doable, but it will require non-trivial effort to implement and validate. The d4xx and serdes drivers diverged quite a bit between JP4 and JP5.

Not the "Dual" for JP4 but d4xx/serdes development. i can merge this feature but all commits with history will be lost.
I will use same driver for Alderlake, we want to have only one copy of the driver to maintain.

We also have issue with y12i on viewer on JP4 and it resolved on dual version.

**Comment by jnzw** [comment]
> We also have issue with y12i on viewer on JP4 and it resolved on dual version.

That's strange. There was an issue with Y12I indeed but it was with JP5 and it should have been fixed by #145.  I don't think I had issue with y12i with JP4 before doing porting.

**Pull request merged**